airbyte-integrations/connectors/source-hubspot/integration_tests/abnormal_state.json

```diff
@@ -190,7 +175,7 @@
     "type": "STREAM",
     "stream": {
       "stream_descriptor": { "name": "tickets" },
-      "stream_state": { "updatedAt": "2221-10-12T13:37:56.412000+00:00" }
+      "stream_state": { "updatedAt": "2200-01-01T00:00:00.000000Z" }
```

Comment:
@maxi297 I think this might be a format change, thus should be breaking, correct me if i'm wrong @maxi297 @ChristoGrab

Reply:
I guess it depends: can the new code take both format as an input? If not, this is breaking and requires a reset

Reply:
Looks like it can, because the CAT incremental tests will take the new format as an input, thus `passes`.

Reply:
Hey team! So I made this change because the `future_state` tests were complaining that the format of state provided didn't match the expected pattern (the offending stream was not specified). Looking at the state object from the live tests, it seems different streams have different formats, I'm seeing a mix of both `00+00` and `00.000` patterns. Either way this is just changing the state object provided for testing, so unless I'm mistaken this shouldn't actually have any effect outside the incremental test run.